controllers/templatesync/template_sync.go

```diff
@@ -828,13 +1018,15 @@ func (r *PolicyReconciler) handleSyncSuccess(
 	tName string,
 	msg string,
 	resInt dynamic.ResourceInterface,
+	gvr *schema.GroupVersionKind,
 ) error {
 	if msg != "" {
 		r.Recorder.Event(pol, "Normal", "PolicyTemplateSync", msg)
 	}
 
-	// Only do additional steps if a template-error is the most recent status
-	if !strings.Contains(getLatestStatusMessage(pol, tIndex), "template-error;") {
+	// Only do additional steps if a template-error is the most recent status or this isn't an OCM policy
+	if gvr.Group != "policy.open-cluster-management.io" ||
```

Comment:
I actually have this one locally--not sure how it didn't get pushed yet. I'm using `common.APIGroup`, but this one might be more accurate. Do you have a preference?

Reply:
I went with `policiesv1`. Probably doesn't make much difference here, but in other places it makes more sense to use `common.APIGroup` because it's a `const` used inside of `const` strings for the label keys.